setup.py

```diff
@@ -0,0 +1,29 @@
+"""Setup file.
+"""
+__license__ = "Apache License"
```

Comment:
We have not decided on a license yet; please remove this.

Reply:
Give me a sec, I will update it.

Reply:
Done.